### PR TITLE
Delete IBM seq2seq.

### PR DESCRIPTION
**Pull request by stephenroller**
Leaving this up to track some line count stats.

starter pack

### CODE REVIEW

**Comment by emilydinan** [comment]
😮 😮 😮

**Comment by stephenroller** [comment]
Let's go ahead with this as is.

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,2 +0,0 @@
-sphinx
-sphinx_rtd_theme
```
Comment (hadasah) [inline docs/requirements.txt]: what's the purpose of this?

Comment (stephenroller) [reply docs/requirements.txt]: Vestigial file which should've been deleted in #1544.

**Pull request merged**